packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -78,9 +80,25 @@ public void updateBuilder(@NonNull CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-    requestBuilder.set(
-        CaptureRequest.CONTROL_AE_REGIONS,
-        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+
+    MeteringRectangle[] exposureRegions = null;
+
+    if (exposureRectangle != null) {
+      exposureRegions = new MeteringRectangle[] {exposureRectangle};
+    } else if (!defaultRectangleHasBeenSet) {
+      exposureRegions =
+          new MeteringRectangle[] {
+            new MeteringRectangle(0, 0, 1, 1, MeteringRectangle.METERING_WEIGHT_MAX)
+          };
+      defaultRectangleHasBeenSet = true;
+    }
+
+    try {
+      requestBuilder.set(CaptureRequest.CONTROL_AE_REGIONS, exposureRegions);
+    } catch (Exception e) {
```

Comment:
Not sure we need this if we ensure `exposureRegions` is not null by using a default value.